docs/streams/upgrade-guide.html

```diff
@@ -76,8 +76,13 @@ <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API
 
     <!-- Placeholder KIP-213 -->
     <!-- Placeholder KIP-307 -->
-    <!-- Placeholder KIP-479 -->
+    <p>
+        Before the 2.4.0 version of Kafa Streams, users of the DSL could not name the state stores involved in a Stream-Stream join.  If users added DSL operations upstream of the 
```

Comment:
```suggestion
        Before the 2.4.0 version of Kafa Streams, users of the DSL could not name the state stores involved in a Stream-Stream join.  If users added DSL operations before the 
```

Super subtle difference, but the only thing that affects the names is the order in which you call the DSL operators, so you can actually add upstream operations, as long as you add the new DSL calls after the original ones. I wouldn't get into it in the docs, but it's easy to swap "before" for "upstream" and be technically correct.

Reply:
ack